python/spacewalk/server/dbConfig.py

```diff
@@ -0,0 +1,64 @@
+#  pylint: disable=missing-module-docstring,invalid-name
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025 SUSE LLC
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+from . import rhnSQL
+
+
+def value(key):
+    """
+    Return the value of the given key. If the value is not defined, return the default value
+    When the key is not found, an AttributeError is raised
+    """
+
+    def sanitize_value(key, val):
```

Comment:
If we move this out of the `value()` function, it's easier to test this function.